README.md

```diff
@@ -6,14 +6,15 @@
 
 ## Windows
 
-Once you've cloned the repo locally on your machine, open a command line/power shell in that folder and run ```yarn``` or ```npm install``` to install the packages for the app.
+Make sure you download and install NodeJS with the version between 12.X and 14.X, we recommend 14.17.2. The latest versions are not supported.
+
+Once you've cloned the repo locally on your machine, open a command line/power shell in that folder and run ```yarn``` to install the packages for the app.
 
 Next up you need to set the environment variables for Android on your local machine, you can find the instructions on how to do that [here](https://levelup.gitconnected.com/android-react-native-window-setup-how-to-setup-android-environment-for-react-native-app-588aaa13c3a6).
```

Comment:
usually it's better to find official documentation, in case of android from google, this will be more uptodate for all platforms. If you cannot find one from google or react native, it's fine